apps/ui/lib/store/actioncreators/index.ts

```diff
@@ -550,7 +522,6 @@ export const actionCreators = {
 						dispatch(actionCreators.setTypes(types));
 						dispatch(actionCreators.setRelationships(relationships));
 						dispatch(actionCreators.setOrgs(orgs));
-						dispatch(actionCreators.setGroups(groups, user));
```

Comment:
Do we use `setGroups` anywhere else?

Reply:
We do in [/jellyfish/apps/ui/lib/components/ChatWidget/components/Layout.tsx](https://github.com/product-os/jellyfish/blob/ff0799440f08020422d62b12dce64796aebcca96/apps/ui/lib/components/ChatWidget/components/Layout.tsx#L66)

Reply:
I think that uses a seperate redux store in the Chat widget component